packages/atlas/src/api/queries/videos.graphql

```diff
@@ -91,8 +91,8 @@ query GetVideos($offset: Int, $limit: Int, $where: VideoWhereInput, $orderBy: Vi
   }
 }
 
-query GetBasicVideos($where: VideoWhereInput) {
-  videos(where: $where) {
+query GetBasicVideos($limit: Int, $where: VideoWhereInput) {
+  videos(limit: $limit, where: $where) {
     ...BasicVideoFields
```

Comment:
This currently doesn't provide all the fields required by the `VideoTile`. You can see that this has the effect that once this query finishes, separate `GetVideo` is fired for each video to get missing details (check out network tab). We will need to add some missing fields here to make sure subsequent requests are not required